src/Platforms/MariaDb1043Platform.php

```diff
@@ -81,17 +82,22 @@ public function getColumnTypeSQLSnippet(string $tableAlias = 'c', ?string $datab
             return parent::getColumnTypeSQLSnippet($tableAlias, $databaseName);
         }
 
+        // 't' alias is already in use for following query
+        if ($tableAlias === 't') {
+            throw new InvalidArgumentException('Table alias "t" is not allowed, please choose another one.');
+        }
```

Comment:
Maybe we can just pick anther one in that case? Disallowing the value `'t'` feels a bit arbitrary.

Reply:
You're right, should now be a better solution.